dev/service-registry-client/src/main/scala/com/lightbend/lagom/internal/registry/ServiceRegistryModule.scala

```diff
@@ -34,13 +34,6 @@ class ServiceRegistryModule(environment: Environment, configuration: Configurati
           "bound to an implementation that will query the embedded Service Locator. This is fine to use " +
           "only during development."
       }
-    } else if (environment.mode == Mode.Test) {
```

Comment:
Alright, I think I agree. It's a test concern, so it shouldn't be in the "production" code. My reason for having it here was users conveniency, as users wouldn't have to bind a service locator for their "single-service" tests (and they could still override this default with Module.override). But I guess we can provide a test helper for achieving the same. Al good for me.